File: Exercicios-Beecrowd/Kotlin/1044.kt
import java.util.*

fun main(args: Array<String>) {
    val num= readLine().toString().split(" ")
    val a=num[0].toInt()
    val b=num[1].toInt()
    if (a%b==0 || b%a==0){
        println("Sao Multiplos")

    }
   else{
       println("Nao sao Multiplos")
    }
	
}

File: Exercicios-Beecrowd/Python/1044.py
x,y= input().split(" ")
x = int(x)
y= int(y)
if(y%x== 0)or(x%y== 0): print('Sao Multiplos')
if(y%x!=0)and(x%y!=0):print('Nao sao Multiplos')